docs/example-scenario/update_social_images.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
Can we move this file to the /build folder?

Reply:
Done.